docs/app/Examples/modules/Dropdown/Types/Trigger.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Dropdown, Icon } from 'stardust'
+
+const trigger = (
+  <div style={{display: 'inline-block'}}>
+    <Icon name='user' />
+    Hello, Bob
+  </div>
+)
+
+const DropdownExample = () => (
+  <Dropdown trigger={trigger}>
+    <Dropdown.Menu>
+      <Dropdown.Item text='New' />
```

Comment:
You may already be doing this, but how about shortening this Menu to only a couple common user menu options?  Like settings, logout, etc.